code/modules/reagents/chemistry/recipes/medicine.dm

```diff
@@ -226,3 +226,18 @@
 	id = "corazone"
 	results = list("corazone" = 3)
 	required_reagents = list("phenol" = 2, "lithium" = 1)
+
+/datum/chemical_reaction/morphine
+	name = "Morphine"
+	id = "morphine"
+	results = list("morphine" = 3)
+	required_reagents = list("carbon" = 2, "hydrogen" = 2, "nitrogen" = 1, "oxygen" = 1)
+	required_temp = 480
```

Comment:
Did you mean to include this?

Reply:
No, I am a double dumbfuck for first pushing this entire thing to the morphine PR, and now for leaving the morphine thing in this, apologies